### ISSUE TITLE
There is no 4-sided visible focus on dynamic page sub header 

### ISSUE THREAD

**Issue by nagbhushanr** [SAP/fundamental-ngx#9086]
#### Is this a bug, enhancement, or feature request?
Accessibility Issue

#### Briefly describe your proposal.
**Observed Behavior:**
There is no 4-sided visible focus provided to ‘Quick Start’ heading region.

**Expected Behavior:**   
All UIs should be provided with sided visible focus. 

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
angular - 13.3.7
fundamental ngx - 0.36.0

#### If this is a bug, please provide steps for reproducing it.

https://sap.github.io/fundamental-ngx/fundamental-ngx#/core/dynamic-page
Inspect and check for dynamic page subheader 

#### Is there anything else we should know?

![image](https://user-images.githubusercontent.com/119582923/207225009-e86604ab-8a9d-4970-b861-dd27922f21ef.png)

**Comment by N1XUS** [comment]
@nagbhushanr Could you please explain why would you like to have visible focus state for the element which is not interactable?

**Comment by nagbhushanr** [comment]
Hi @N1XUS this is the accessibility issue reported by the team.... so basic idea is elements like this should have a 4 sided visible focus in order to provide the assistance to people with low vision I believe
